code/game/objects/structures/watercloset.dm

```diff
@@ -446,21 +569,100 @@
 	if(!(istype(O)))
 		return
 
+	if(iswrench(O))
+		var/obj/item/weapon/wrench/W = O
+
+		var/choices = list()
+		if(anchored)
+			choices += "Wash"
+			if(can_move)
+				choices += "Disconnect"
+		else
+			choices += "Connect"
+			if(can_rotate)
+				choices += "Rotate"
+
+		var/response = input(user, "What do you want to do?", "[src]") as null|anything in choices
+		if(!Adjacent(user) || !response)	//moved away or cancelled
+			return
+		switch(response)
+			if("Wash")
+				busy = 1
+				var/wateract = 0
+				wateract = (W.wash(user, src))
+				busy = 0
+				if(wateract)
+					W.water_act(20,310.15,src)
+			if("Disconnect")
+				user.visible_message("<span class='notice'>[user] is disconnecting [src].</span>", "<span class='notice'>You are disconnecting [src]...</span>")
```

Comment:
Tenses. `"<span class='notice'>[user] starts disconnecting [src]...</span>", "<span class='notice'>You start disconnecting [src]...</span>"`